app/components/container/form-scale/component.js

```diff
@@ -31,6 +31,7 @@ const DEFAULT_CONFIG = {
     failedJobsHistoryLimit: HISTORY_LIMIT,
     schedule: '0 * * * *',
     successfulJobsHistoryLimit: HISTORY_LIMIT,
+    jobConfig: {},
```

Comment:
(I realize that statefulSet has an existing example of this just above with an array, but) this is bad.. all instances created from this are going to share the same `jobConfig` object because there's only one object instance, defined here.  And createRecord() does not do a deep copy.